lib/metasploit/framework/data_service/stubs/note_data_service.rb

```diff
@@ -1,7 +1,19 @@
 module NoteDataService
 
+  def notes(opts)
+    raise 'NoteDataService#notes is not implemented'
```

Comment:
`raise NotImplementedError` I think is what has generally been used for this kind of thing. This applies to the rest of this file as well.

Reply:
Thanks, nice catch. I'll add a ticket to our backlog to address the other modules under 
 `lib/metasploit/framework/data_service/stubs/`.